tests/default/bad/empty.catala_en

```diff
@@ -19,8 +19,7 @@ $ catala test-scope A
 6 │   output y content boolean
   │          ‾
   └─ Article
-[ERROR] This variable evaluated to an empty term (no rule that defined it
-  applied in this situation)
+[ERROR] Error during evaluation: no computation with valid conditions found.
```

Comment:
The new error message is more cryptic than the old one, plus we should really be explicit and learning-friendly with this precise error message as this is one that newcomers are going to encounter a lot and early.